app/helpers/application_helper.rb

```diff
@@ -61,4 +61,40 @@ def jquery_include_tag
     buf << [ javascript_tag("$.fx.off = true;") ] if Rails.env.test?
     buf.join("\n").html_safe
   end
-end
+
+  # Replicates rails's [current_page?] method, so it respects the format appended to the url 
+  # We should remove this method when [rails] update it.
+  #
+  # Let's say we're in the <tt>http://www.diaspora-pod-example.com/aspects.mobile</tt>
+  # diaspora_current_page?(:aspects)
+  # => true
+  #
+  def diaspora_current_page?(options)
```

Comment:
Assignment Branch Condition size for diaspora_current_page? is too high. [31.3/15]